### ISSUE TITLE
ts-jest warnings

### ISSUE THREAD

**Issue by ssaarela** [finnair/v-validation#44]
Running tests logs warnings:
```
ts-jest[config] (WARN) There is a mismatch between your NodeJs version v12.13.0 and your TypeScript target esnext. This might lead to some unexpected errors when running tests with `ts-jest`. To fix this, you can check https://github.com/microsoft/TypeScript/wiki/Node-Target-Mapping
```
We should probably fix these, but how to do that while still continuing to support both NodeJs versions 10 and 12?
